src/Microsoft.ServiceFabric.Actors/Runtime/ActorManager.cs

```diff
@@ -1254,7 +1254,7 @@ private async Task LoadRemindersAsync(CancellationToken cancellationToken)
                 actorReminder,
                 (k, v) =>
                 {
-                    v.CancelTimer();

```

Comment:
CancelTimer call does Dispose internally, could you please explain a bit more how this Dispose here helps?

Reply:
the logic is the same the only difference is that on Dispose we call - GC.SuppressFinalize
and we have extreme case - of ~20k actors we reminders that are triggered once in ~5 minutes and each time inside  ReceiveReminderAsync we are - calling RegisterReminderAsync - this causes some stress on the finalization thread handling spikes of ActorReminders finilizers